Okta.Core.Automation/Factors/ActivateOktaUserFactor.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System.Management.Automation;
+using Okta.Core.Models;
+
+namespace Okta.Core.Automation
+{
+    [Cmdlet("Activate", "OktaUserFactor")]
+    public class ActivateOktaUserFactor : OktaCmdlet
+    {
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 0,
+            HelpMessage = "Id or username of the Okta user"
+        )]
+        public string IdOrLogin { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 1,
+            HelpMessage = "Id of the user factor to remove. If empty, removes all user factors"
```

Comment:
Double checking - since it's mandatory, will it work if it's empty or not specified?